earth_enterprise/BUILD_RHEL_Centos.md

```diff
@@ -0,0 +1,116 @@
+# GEE Build Setup for CentOS, RHEL 6 and RHEL 7     
+
+## Enable additional repos
+Different yum repositoies need to be enabled for different distributions to install several required development tools.  
+
+### CentOS 7 
+```
+sudo yum install epel-release
+```
+
+### RHEL 7  
+
+```
+sudo subscription-manager repos --enable=rhel-7-server-extras-rpms
+sudo subscription-manager repos --enable=rhel-7-server-optional-rpms
+sudo yum install -y epel-release
+```    
+
+### RHEL 6
+```
+sudo subscription-manager repos --enable=rhel-x86_64-workstation-dts-2
+sudo subscription-manager repos --enable=rhel-6-server-optional-rpms
+```
+
+## Install git  
+    
+Install the system default version:
+    
+```
+sudo yum install -y git
```

Comment:
This doesn't work for me on RHEL 6.  The default version of git is too old to use git lfs.  The ius repo version below works fine as long as you adjust it for rhel6 instead of rhel7.

Reply:
Also, if git happens to be installed before you install it from the ius repo you have to uninstall it first.  I'm not sure if that's worth mentioning: I ran into it because I installed RHEL6 as a developer workstation, so it installed git automatically, but most people probably won't run into that.

Reply:
Maybe we just leave it as is.  We have the option to update it in the future if we determine that there's a problem.

Reply:
Double checked - yep, having installed epel, git lfs works fine.